### PR TITLE
[stable/unbound] add clusterIP

### PR DESCRIPTION
**Pull request by n0rad**
Add clusterIP value to unbound service

### CODE REVIEW

**Comment by n0rad** [comment]
I'm using unbound as a forward server for coredns and it can only be set by IP.
So to stay in the cluster, setting clusterIP is required.

@scottrigby can you review the PR please ?

**Comment by n0rad** [comment]
@viglesiasce, ca you review the PR ?

**Comment by n0rad** [comment]
up

**Comment by n0rad** [comment]
up

**Comment by scottrigby** [comment]
This chart doesn't appear to be maintained. No OWNERS file, one of the containers googlecontainer/exechealthz repo doesn't  exist on docker hub anymore, so this doesn't install with default values (the source code repo was also retired: https://github.com/kubernetes-retired/contrib/tree/master/exec-healthz). Is this a good candidate for sunsetting?

**Comment by zanhsieh** [comment]
@scottrigby @n0rad 
For me, yes, it's a good candidate for sunsetting, unless:
- The contributor(s) find other repo for hosting container.
- Find new chart owners.

I hope even chart sunsetting, it won't get deleted. Otherwise it will take a lot of efforts to climb back from incubator to stable.

**Comment by WyriHaximus** [comment]
> * Find new chart owners.

Where do I sign up for that?



> This chart doesn't appear to be maintained. No OWNERS file, one of the containers googlecontainer/exechealthz repo doesn't exist on docker hub anymore, so this doesn't install with default values (the source code repo was also retired: https://github.com/kubernetes-retired/contrib/tree/master/exec-healthz). Is this a good candidate for sunsetting?

Currently working on preparing a PR on fixing that dependency, also bumping the version of unbound, and also do some other small fixes in the chart it self like #18569 .

File: stable/unbound/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Unbound is a fast caching DNS resolver
 home: https://www.unbound.net/
 name: unbound
-version: 1.0.0
+version: 1.0.1
```
Comment (davidkarlsen) [inline stable/unbound/Chart.yaml]:
```suggestion
version: 1.1.0
```

**Comment by zanhsieh** [comment]
@n0rad 
```
I1106 07:55:44.573]   Warning  Failed     9m42s (x2 over 9m58s)  kubelet, gke-jenkins-default-pool-51cafeba-9gxc  Failed to pull image "googlecontainer/exechealthz:1.2": rpc error: code = Unknown desc = Error response from daemon: pull access denied for googlecontainer/exechealthz, repository does not exist or may require 'docker login'
I1106 07:55:44.573]   Warning  Failed     9m42s (x2 over 9m58s)  kubelet, gke-jenkins-default-pool-51cafeba-9gxc  Error: ErrImagePull
```

**Comment by WyriHaximus** [comment]
@n0rad I'm also looking into the specific error that @zanhsieh mentioned. Are you also doing that or should I continue?

**Comment by n0rad** [comment]
@WyriHaximus I'm not and was looking for your PR/issue, but I suppose you did not submitted it yet.

**Comment by WyriHaximus** [comment]
@n0rad haven't submitted it yet as I was intending to it in a PR that also bumps the image and some other stuff. But fixing this issue turned out a bit harder then I thought. I'll submit an issue about this issue and also mention that I'm working on a fix so if anyone else bumps into it they can more easily find out. :+1:

**Comment by n0rad** [comment]
I added a commit to fix the exechealthz container that do not exists anymore and fixed the deprecated deployment app version that is not compatible with new versions of kubernetes.

I can join the maintainers if you need some.

**Comment by WyriHaximus** [comment]
Sweet! This also resolves #18664 and closes #18683 👍

**Pull request merged**